recipes/ckanweb-deploy-exts.rb

```diff
@@ -259,7 +260,8 @@
 			cwd "#{config_dir}"
 			code <<-EOS
 				if [ -z "$(grep 'ckanext.resource_visibility:schema/presets.json' production.ini)" ]; then
-					sed -i "s/scheming.presets = ckanext.scheming:presets.json ckanext.data_qld:presets.json$/scheming.presets = ckanext.scheming:presets.json ckanext.data_qld:presets.json ckanext.resource_visibility:schema/presets.json/g" production.ini;
+					# scheming.presets = ckanext.scheming:presets.json ckanext.data_qld:presets.json ckanext.resource_visibility:schema/presets.json
+					sed -i "s/ckanext.data_qld:presets.json$/ckanext.data_qld:presets.json ckanext.resource_visibility:schema/presets.json/g" production.ini;
```

Comment:
This is a step in the right direction, but what if another extension became involved and a different entry was at the end of the line? We're already testing to ensure that the visibility presets didn't exist, so there's probably no need for the `$` here.

Reply:
removed